WordPress/Classes/ViewRelated/Post/Scheduling/SchedulingCalendarViewController.swift

```diff
@@ -1,253 +0,0 @@
-import Foundation
-import Gridicons
-
-// MARK: - Date Picker
-@available(iOS, deprecated: 14.0, message: "Use SchedulingDatePickerViewController, based on UIDatePicker.inline")
-class SchedulingCalendarViewController: UIViewController, CalendarSheet, DateCoordinatorHandler, SchedulingViewControllerProtocol {
```

Comment:
The deprecation annotation recommends using `SchedulingDatePickerViewController` instead of `SchedulingCalendarViewController` in iOS 14 and above. 

Now that we support only iOS 14 and above, this whole file can be deleted. Notice that in this file we also had `CalendarSheet` and `TimePickerViewController`.